catalog/resource_metastore.go

```diff
@@ -76,8 +76,8 @@ func ResourceMetastore() *schema.Resource {
 				Type:     schema.TypeBool,
 				Optional: true,
 			}
-			m["delta_sharing_enabled"].RequiredWith = []string{"delta_sharing_recipient_token_lifetime_in_seconds"}
-			m["delta_sharing_recipient_token_lifetime_in_seconds"].RequiredWith = []string{"delta_sharing_enabled"}
+			m["delta_sharing_scope"].RequiredWith = []string{"delta_sharing_recipient_token_lifetime_in_seconds"}
```

Comment:
Can you add validation for the possible values of `delta_sharing_scope` using the `m["delta_sharing_scope"].ValidateFunc = validation.StringInSlice([]string{...})` ?

Reply:
Done